unit-9/exercise-3.js

```diff
@@ -0,0 +1,7 @@
+const inNumber = (int) => {
```

Comment:
compareTo100
Hoặc
compareNumber

Không đặt tên biến là `int` đọc gây lú
```suggestion
const isGreaterThan100 = num => {
```